mypy/build.py

```diff
@@ -128,6 +128,8 @@ def build(sources: List[BuildSource],
           alt_lib_path: Optional[str] = None,
           flush_errors: Optional[Callable[[List[str], bool], None]] = None,
           fscache: Optional[FileSystemCache] = None,
+          stdout: TextIO = sys.stdout,
+          stderr: TextIO = sys.stderr,
```

Comment:
This fixes the stdout/stderr streams to whatever they are when the module is imported. That doesn't sound ideal. Maybe leave them `None` and dynamically choose, e.g. `file=(stdout or sys.stdout)`. (Note that `print()` lets you say `print(..., file=None)` and it will use sys.stdout. There's no such shortcut for stderr, but it seems it's used less.) Of course a bunch of types will need to be changed from `TextIO` to `Optional[TextIO]`.

Alternatively, if there's a single "toplevel" place where stdout/stderr are passed in, you could set the default to `None` only there, and add `if not stdout: stdout = stdout` and ditto for `stderr`.

(Basically, there are two options -- turn `None` into `sys.stdout` as early as possible (but not at module load time, i.e. not as a default value), or do it as late as possible (in the `print()` calls). Your choice.